images/build/setcap/variants.yaml

```diff
@@ -1,6 +1,5 @@
 variants:
-  # Debian 11 - Kubernetes 1.23 and newer
-  bullseye:
```

Comment:
should we drop this? are we planning to update all release branches with the new one? otherwise i would keep this and update the comments

Reply:
Good, readded the Debian 11 variants.